### ISSUE TITLE
SVHighlight width spans outside of parent container

### ISSUE THREAD

**Issue by arkmech** [bennymi/svhighlight#7]
<img width="1064" alt="Screenshot 2024-06-29 at 07 47 26" src="https://github.com/bennymi/svhighlight/assets/114366662/efff7607-41ec-4e55-acdd-79a615a02c5f">

I have looked in Chrome Dev Tools and cannot determine what is causing the SVHighlight to span outside of div.

I do not want to set a max width, but more of width 100% within its parent container.

The area outside the gray is outside the parent.

**Comment by bennymi** [comment]
Hi @arkmech, can you provide a reproduction for it or a code snippet so I can see how you are implementing the code block?

**Comment by bennymi** [comment]
Btw this library hasn't been updated in quite a long time and there are now some better solutions than highlight.js, so you might consider trying out [shiki](https://shiki.matsu.io/guide/dual-themes) instead. It also shouldn't be too difficult to set up.

**Comment by arkmech** [comment]
> Hi @arkmech, can you provide a reproduction for it or a code snippet so I can see how you are implementing the code block?

```ts
<script lang='ts'>
  import { CodeBlock } from 'svhighlight';

  interface CodeSnippetProps {
    code?: string
  }
  let { code, }: CodeSnippetProps = $props();
</script>

{#if code}
  <CodeBlock 
    language="svelte"
    {code}
    headerClasses="bg-secondary dark:bg-secondary-dark border-b border-secondary dark:border-secondary-dark font-semibold text-sm text-primary dark:text-primary-dark [&>button]:text-secondary dark:[&>a]:text-secondary-dark hover:[&>button]:text-secondary-hover dark:hover:[&>a]:text-secondary-hover-dark"
    background="bg-transparent border border-secondary dark:border-secondary-dark"
    codeTextClasses="text-utility-gray-700"
    lineNumberTextClasses='bg-secondary dark:bg-secondary-dark text-quarterary dark:text-quarterary-dark border-r'
  />
{/if}
```

**Comment by arkmech** [comment]
> Btw this library hasn't been updated in quite a long time and there are now some better solutions than highlight.js, so you might consider trying out [shiki](https://shiki.matsu.io/guide/dual-themes) instead. It also shouldn't be too difficult to set up.

Yeah, didn't look at commit history. But yeah how I style was just copying highlight.js style and modifying the colors of one theme.
